2019/5xxx/CVE-2019-5615.json

```diff
@@ -1,18 +1,121 @@
+                                 "version_name": "6.5.11",
+                                 "version_value": "6.5.11"
+                              },
+                              {
+                                 "version_affected": "!>=",
+                                 "version_name": "6.5.50",
+                                 "version_value": "6.5.50"
+                              }
+                           ]
+                        }
+                     }
+                  ]
+               },
+               "vendor_name": "Rapid7"
             }
-        ]
-    }
-}
\ No newline at end of file
+         ]
+      }
+   },
+   "credit": [
+      {
+         "lang": "eng",
+         "value": "This issue was reported privately to Rapid7 by Robert Elliott from IBM Security."
+      }
+   ],
+   "data_format": "MITRE",
+   "data_type": "CVE",
+   "data_version": "4.0",
+   "description": {
+      "description_data": [
+         {
+            "lang": "eng",
+            "value": "Users with Site-level permissions can access files containing the username-encrypted passwords of Security Console Global Administrators and clear-text passwords for restoring backups, as well as the salt for those passwords. Valid credentials are required to access these files and malicious users would still need to perform additional work to decrypt the credentials and escalate privileges. This issue affects:\nRapid7 InsightVM version 6.5.49 and prior versions.\n"
+         }
+      ]
+   },
+   "generator": {
+      "engine": "Vulnogram 0.0.6"
+   },
+   "impact": {
+      "cvss": {
+         "attackComplexity": "HIGH",
+         "attackVector": "NETWORK",
+         "availabilityImpact": "NONE",
+         "baseScore": 3.1,
+         "baseSeverity": "LOW",
+         "confidentialityImpact": "LOW",
+         "integrityImpact": "NONE",
+         "privilegesRequired": "LOW",
+         "scope": "UNCHANGED",
+         "userInteraction": "NONE",
+         "vectorString": "CVSS:3.0/AV:N/AC:H/PR:L/UI:N/S:U/C:L/I:N/A:N",
+         "version": "3.0"
+      }
+   },
+   "problemtype": {
+      "problemtype_data": [
+         {
+            "description": [
+               {
+                  "lang": "eng",
+                  "value": "CWE-257"
+               }
+            ]
+         },
+         {
+            "description": [
+               {
+                  "lang": "eng",
+                  "value": " Storing Passwords in a Recoverable Format"
+               }
+            ]
+         }
+      ]
+   },
+   "references": {
+      "reference_data": [
+         {
+            "name": "https://help.rapid7.com/insightvm/en-us/release-notes/#6.5.50",
+            "refsource": "CONFIRM",
+            "url": "https://help.rapid7.com/insightvm/en-us/release-notes/#6.5.50"
+         }
+      ]
+   },
+   "solution": [
+      {
+         "lang": "eng",
+         "value": "Update to version 6.5.50 or later."
```

Comment:
Specifying component to update: “Update your Security Console to version 6.5.50 or later."